src/dev/i18n/integrate_locale_files.ts

```diff
@@ -206,19 +215,94 @@ async function writeMessages(
   }
 }
 
+function isSpreadableObject(value: unknown) {
+  try {
+    return { ...value }, true;
```

Comment:
i think `[...null]` results in error but `{...null}` works, should it consider `null` as non spreadable?

reading a bit more seems it's iterable vs enumerable, looking at usage i think it's fine since it's spreading into an object

Reply:
`null` is an acceptable argument but thanks for thinking about it.